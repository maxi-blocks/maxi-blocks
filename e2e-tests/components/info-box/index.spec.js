/**
 * WordPress dependencies
 */
import { createNewPost } from '@wordpress/e2e-test-utils';

/**
 * Internal dependencies
 */
import {
	changeResponsive,
	insertMaxiBlock,
	updateAllBlockUniqueIds,
} from '../../utils';

describe('InfoBox', () => {
	it('Check Infobox', async () => {
		await createNewPost();
		await insertMaxiBlock(page, 'Text Maxi');
		await updateAllBlockUniqueIds(page);
		await changeResponsive(page, 's');

		const warningBox = await page.$eval(
			'.components-panel .maxi-warning-box',
			warning => warning.innerHTML
		);

		expect(warningBox).toMatchSnapshot();
	});
});
