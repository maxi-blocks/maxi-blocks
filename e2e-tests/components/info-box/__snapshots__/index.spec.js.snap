// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoBox Check Infobox 1`] = `"Responsive mode activated. Edits apply exclusively to selected break points. Return to \"Your size\" for global editing (recommended)."`;
